package.json

```diff
@@ -142,7 +143,7 @@
     "@reach/auto-id": "^0.18.0",
     "@restart/ui": "^1.0.1",
     "@tailwindcss/forms": "^0.5.0",
-    "@yext/analytics": "^0.6.4",
+    "@yext/analytics": "^0.5.0",
```

Comment:
should we be downgrading?

Reply:
upon further investigation, downgrading doesn't seem necessary. while were here, i'll bump to the latest v0.6.5